framework/db/ColumnSchemaBuilder.php

```diff
@@ -115,6 +115,26 @@ class ColumnSchemaBuilder extends Object
      * @since 2.0.8
      */
     public $comment;
+    /**
+     * @var string the name of the foreign key constraint.
+     */
+    public $foreignKeyName;
```

Comment:
is it ok to make them public and not protected since only `foreignKey()` method can modify them